source/reference/mongodb-wire-protocol.txt

```diff
@@ -121,7 +121,7 @@ Request Opcodes
 ~~~~~~~~~~~~~~~
 
 .. note::
-   Starting with MongoDB 2.6 and :data:`~isMaster.maxWireVersion` ``3``,
+   Starting with MongoDB 2.6 and :data:`~hello.maxWireVersion` ``3``,
```

Comment:
• We need a blank line between the `note` directive and its contents. While this would build fine on OldGen, this in fact breaks on NextGen.